app/Components/CreateNew.js

```diff
@@ -0,0 +1,28 @@
+import { View, Image, StyleSheet } from 'react-native';
+
+export default function CreateNew() {
+  return (
+    <View style={styles.container}>
+      <Image
+        source={require('../assets/add-sign.png')}
+        style={styles.image}
+      />
+    </View>
+  )
+}
+
+const styles = StyleSheet.create({
+  image: {
+    width: 55,
+    height: 55,
+  },
+  container: {
+    backgroundColor: 'rgba(0, 0, 0, 0.2)',
+    display: 'flex',
```

Comment:
React Native has `display: flex;` by default, so you don't need this line of code.